kolibri/plugins/management/assets/src/device_management/views/welcome-modal.vue

```diff
@@ -0,0 +1,73 @@
+<template>
+
+  <core-modal
+    :title="$tr('welcomeModalHeader')"
+    :enableBgClickCancel="false"
+    @cancel="emitCloseModal"
+  >
```

Comment:
I would have liked to make this a separate component to separate the translations, but I thought it'd be more appropriate in a separate PR